src/pyuvsim/uvsim.py

```diff
@@ -786,11 +792,37 @@ def run_uvdata_uvsim(
     if not isinstance(input_uv, UVData):
         raise TypeError("input_uv must be UVData object")
 
+    if beam_list.beam_type != "efield":
```

Comment:
The codecov report says the repo remains 100% covered, but the file-by-file view suggests there are a number of lines (14 in total) here in uvsim.py that aren't covered by tests.

Reply:
Yeah, these annotations are not accurate and are really annoying. Codecov is actually deprecating them because there's a limitation in the GitHub API that prevents codecov from updating or deleting the annotations after more CI jobs have run and the GitHub API also won't let codecov tell when all the runs have finished, so codecov cannot just wait until they're all done. Details here: https://docs.codecov.com/docs/github-checks

I could turn them off by adding a `.codecov.yml` file or we can just wait until codecov kills them. You can also toggle them off when you're reading a PR by pressing `a`.